### PR TITLE
remove delete block in global page if used in other page

### PR DESCRIPTION
**Pull request by alavieille**
[OO-FEATURE] It is no longer possible to delete a block of global page if it is used

https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1599
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/565

### CODE REVIEW

File: ApiBundle/Tests/Transformer/BlockTransformerTest.php [hunk 1]
```diff
@@ -147,6 +148,54 @@ public function blockTransformProvider()
     }
 
     /**
+     * @param $areas
+     * @param $nodeId
+     * @param $nodeMongoId
+     * @param $isInside
+     * @param $isDeletable
```
Comment (KevinArtus) [inline ApiBundle/Tests/Transformer/BlockTransformerTest.php]: What's the type of params ? Boolean, array, String... ?

**Comment by itkg-ngilain** [comment]
:+1: when phpdoc is updated

**Comment by alavieille** [comment]
@KevinArtus  pr updated

**Comment by KevinArtus** [comment]
:+1:

**Pull request merged**